_posts/2020-06-18-Metal3-dev-env-BareMetal-Cluster-Deployment.md

```diff
@@ -371,10 +373,16 @@ sh ${metal3-dev-env-path}/scripts/deprovision/worker.sh
 sh ${metal3-dev-env-path}/scripts/deprovision/controlplane.sh
 sh ${metal3-dev-env-path}/scripts/deprovision/cluster.sh
 ```
-> Note :
+> note "Note"
 > The reason for running the `deprovision/worker.sh` and `deprovision/controlplane.sh` scripts is that not all objects are cleared when we just run the `deprovision/cluster.sh` script. Following this, if you want to deprovision control plane it is recommended to deprovision the cluster itself since we can't provision a new control plane with the same cluster. For worker deprovisioning, we only need to run the worker script.
 
 
+
+This video below exhibits the provisioning and deprovisioning a Kubernetes Cluster as shown above.
```

Comment:
Maybe it's just my non-native English, but I think this sentence is a bit unclear.

An alternative proposal:

> The following video shows all the steps to provision and de-provision a Kubernetes cluster explained above:

But if you prefer the current version (use your judgement :wink: ) I believe it needs at least an _of_: "...exhibits the provisioning ... *of* a Kubernetes...")

Reply:
Thanks for the review!
Fixed, let me know if its looks okay now.